internal/app/siftool/info.go

```diff
@@ -129,6 +129,11 @@ func writeList(w io.Writer, f *sif.FileImage) error {
 			if err == nil {
 				fmt.Fprintf(w, "|%s (%s)\n", dt, f)
 			}
+		case sif.DataOCIBlob:
```

Comment:
Nit: let's keep the white space consistent with the rest of the code.

```suggestion

		case sif.DataOCIBlob:
```